drivers/ethernet/eth_enc28j60.c

```diff
@@ -382,18 +382,18 @@ static void eth_enc28j60_init_mac(struct device *dev)
 	}
```

Comment:
Please change the commit subject to `drivers: eth: enc28j60: Misc fixes`

Reply:
Done